server/service/src/sync/translations/purchase_order_line.rs

```diff
@@ -104,8 +104,10 @@ impl SyncTranslation for PurchaseOrderLineTranslation {
         let result = PurchaseOrderLineRow {
             id,
             purchase_order_id,
-            line_number,
-            item_link_id,
+            line_number: line_number
+                .ok_or_else(|| anyhow::anyhow!("line_number must be present"))?,
+            item_link_id: item_link_id
+                .ok_or_else(|| anyhow::anyhow!("item_link_id must be present"))?,
```

Comment:
Thoughts on this - on OG it seems that these are optional but we're making it not optional on our case. Of course it we can just support the case. But would it be cleaner if it matches up OG?